docker-compose.yml

```diff
@@ -9,9 +9,9 @@ services:
     image: postgres:latest
     command: ["postgres", "-c", "log_statement=all"]
     ports:
-      - 5432:5432
+      - 5434:5432
```

Comment:
**[blocker]** I feel that the `DB_PORT` value in `.env.example` should be updated as well (if we want to keep that file under version control).

Reply:
This is not necessary. DB_HOSTNAME refers to hostname in docker's internal network. Compose creates a new network and attaches containers in the compose-file to this network - containers inside this network can be referenced by container hostname and for accessing services in containers same ports can be used as the services listening inside container.

To sum up 5434:5432 means that service in host machine network will be accessible through host.docker.internal:5434, but inside docker network it is container_hostname:5432

On the other hand I would move the port values that are exposed to host machine network into /.env file (and also methion it in /.env.example) to allow devs to configure their own port values (e.g. when working on multiple projects then ports may overlap). This way this exact change in docker-compose.yml wouldn't be needed as well and /.env.example would provide a good starting point to get it up and running, but leaves free hands for the dev to configure their own ports in case of there is something else listening on that port already

Reply:
@thenouan Yep, good point – if running the application as a docker container, this won’t be an issue. But there’s also the case of running automated tests (commandline `vendor/bin/phpunit`) – in this case, the runtime won’t be Docker, so it won’t have access to the network, right? 

While this is not so important yet, I imagine that if we want to adopt testing in our CI/CD pipeline, then it will become important that we have "unambigous" environment variables for running the tests.

Reply:
@kadmit But in any case, I don’t want to block progress because of this, so let’s consider this **[non-blocker]**. I’ll approve as soon as tests pass with:
```
cp application/.env.example application/.env
docker compose up
cd application && vendor/bin/phpunit
```

Reply:
@DanBondarenko If you run tests inside the container then the same environment is available. If you're running directly on host machine, then yea, network and environment is different. But I would advise to run tests locally on host machine inside container as well to ensure the runtime environment is as close as possible

Reply:
In CI/CD env it is possible to use dind (docker in docker) to setup same environment and to run tests inside the same container instead of setting up the exact same environment as in container to CI/CD machine

Reply:
> I would advise to run tests locally on host machine inside container as well to ensure the runtime environment is as close as possible

@thenouan That is a good recommendation! I’ll follow that from now on – should help avoid issues in the future.

Reply:
Just for information: I attempted running the tests within the container, but since the database needs to be set up and migrated, I was unable to do it (reason appears to be missing env value for config property `pgsql-connection.sync.properties.username`).

<img width="1427" alt="image" src="https://github.com/keeleinstituut/tv-authorization/assets/26142658/be2a6c22-77cd-4f96-b80f-4808d57007b2">

Tests failed (probably same reason as https://github.com/keeleinstituut/tv-authorization/pull/36#discussion_r1221894961)
<img width="963" alt="image" src="https://github.com/keeleinstituut/tv-authorization/assets/26142658/caead725-48c7-4ab1-88af-2df6c18c6a68">